external/common_functions.sh

```diff
@@ -226,14 +226,18 @@ function set_test_info() {
         ubi_packages="git wget unzip"
         ubi_minimal_packages="${ubi_packages}"
         ;;
-    external_custom)
+    ${test})
```

Comment:
I know it's draft but would be better to point it out earlier rather than later.  This is not a good solution.  This is basically saying  if ${test} == ${test}.

Reply:
Yes, I will update this soon by maybe creating an array of supported_external_custom_tests and then when it does not match any of the tests then it will check if any of the tests match the ${test} and if not then use the default error case. I will update it soon.